src/wcontrol/WCzarInfoMap.h

```diff
@@ -114,12 +114,12 @@ class WCzarInfoMap {
     static Ptr create() { return Ptr(new WCzarInfoMap()); }
 
     /// Return the WCzarInfo ptr associated with czId, creating a new one if needed.
-    WCzarInfo::Ptr getWCzarInfo(CzarIdType czId);
+    WCzarInfo::Ptr getWCzarInfo(CzarId czId);
 
 private:
     WCzarInfoMap() = default;
 
-    std::map<CzarIdType, WCzarInfo::Ptr> _wczMap;
+    std::map<CzarId, WCzarInfo::Ptr> _wczMap;
```

Comment:
Inconsistent naming convention `_wczMap` instead of more informative `_wCzarMap`.